app/services/admin/change_induction_service.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Admin
+  class ChangeInductionService
```

Comment:
We might want to make it not work for 2020 cohort - or restrict it to just allow changing CIP perhaps?